us/en/skymiles/styles/styles.css

```diff
@@ -67,6 +67,10 @@
 }
 
 /* Main blocks */
+html {
```

Comment:
This change isn't just for the TOC block. Is it intentional to do it for all of the documents and is just getting included in this PR?

Reply:
Yes, it was intentional. Without it, the smooth scrolling animation doesn't happen from the ToC block to the relevant section.